docs/language/operators.md

```diff
@@ -1094,3 +1095,46 @@ zq -z 'cut District,County | uniq' schools.zson
 {District:"Ojai Unified",County:"Ventura"}
 ...
 ```
+
+---
+
+## `yield`
+
+|                           |                                                 |
+| ------------------------- | ----------------------------------------------- |
+| **Description**           | For each input value, produces one or more values downstream.
+| **Syntax**                | `yield <expression> [, (<expression>)...]` |
+| **Required arguments**    | One or more of `<expression>`, which may be any valid Zed [expression](expressions.md).
+| **Optional arguments**    | None |
+
+#### Example #1:
+
+This example produce two simpler records for every school record listing
+the average math score with the school name and the county name.
+
+```mdtest-command dir=testdata/edu
+zq -Z 'AvgScrMath!=null | yield {school:sname,avg:AvgScrMath}, {county:cname,zvg:AvgScrMath}' testscores.zson
+```
```

Comment:
I know this example is just following the convention established in this file, but I personally find it easier to read examples when I can see the input data, for example:

```
echo '{x:1}' | zq -Z '[the query]' -
```

Reply:
Agreed.  Let's do this in another docs overhaul.  The schools data should be in a tutorial and the reference docs should just have simple, generic examples like `{x:1}` suggested here.